src/main/resources/db/migration/V00000000003__create_table_employee.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE Employee
```

Comment:
No changes required!
Good to know: With the help of spring data dependency this tables can automatically be generated from Entities